### PR TITLE
Let's start with a simple mergebot integration.

### PR DESCRIPTION
**Pull request by orsenthil**
Hey @joerg84  - we need to settle upon the labels for the repo..

* This file is the config for mergebot integration.
* It will apply "requested-changed" as soon as PR is created. I think, we need a different label. What label should it be?

### CODE REVIEW

**Comment by joerg84** [comment]
@orsenthil As discussed. How can I test this PR?

**Comment by orsenthil** [comment]
@joerg84  - once we merge this PR, we give permissions to @mesosphere-mergebot-dev permissions to apply label and it will do something like this (This is test and proof: https://github.com/mesosphere-mergebot/mergebot-test-dcos/pull/144 ) For the config (https://github.com/mesosphere-mergebot/mergebot-test-dcos/blob/master/mergebot-config.json) that is same as the contents of this PR.

File: mergebot-config.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "label": {
+    "enabled": true,
+    "interactive": false,
+    "default": "requested-changes"
```
Comment (joerg84) [inline mergebot-config.json]: This shouldn't be `requested-changes` see here: https://github.com/joerg84/universe/pull/2

**Review by joerg84 — CHANGES_REQUESTED** [review]

**Review by joerg84 — APPROVED** [review]

**Comment by joerg84** [comment]
Thanks Team!

**Pull request merged**